ci/python_bandit_check.sh

```diff
@@ -8,4 +8,5 @@ if [ -e '.git' ]; then
 fi
 
 bandit --format xml -o bandit.xml -r "${gitfiles[@]}" \
+       --exclude utils/rpms/_topdir \
```

Comment:
It looks like this could possibly be also be put in the config file or a .bandit file if we wanted instead.

Reply:
I wonder if we want to make this even more broad:
```suggestion
       --exclude utils/rpms/ \
```

I don't think there would be anything in that dir that we'd want to check.

Reply:
I think that would be entirely reasonable, although if anything did appear that directory I think I'd rather know about it because it could have other consequences.  This whole thing is needed because Jenkins doesn't give you a clean workspace which is a bad idea in the first place IMO.